modules/osdk-multi-arch-building-manifest-list.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-multi-arch-support.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-multi-arch-building-a-manifest-list_{context}"]
+= Building a manifest list to distribute an Operator that supports multiple architectures
+
+You can use a manifest list to build Operators and operands that support multiple architectures. A manifest list points to specific image manifests for one or more architectures. For more information, see link:https://specs.opencontainers.org/image-spec/[OpenContainers Image Specification] or link:https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list[Image Manifest v2, Schema 2].
+
+.Prerequisites
+
+* An Operator project created or maintained with Operator SDK {osdk_ver_n1} or later
+* Docker installed
+
+.Procedure
+
+. For Go-based Operator projects, the Operator SDK explicitly references the `amd64` architecture in your project's Dockerfile. Make the following change to your Dockerfile to set an environment variable to the value specified by the platform flag:
++
+[source,docker]
+----
+GOARCH=$TARGETARCH <1>
+----
+<1> Change the `GOARCH` field from `amd64` to `$TARGETARCH`.
+
+. Run the following command to build your manifest list and push it to your image registry:
++
+[source,terminal]
+----
+$ docker buildx build --push \
```

Comment:
I'm a little confused with this  command.

If i ran the `$ make docker-buildx IMG=<image_registry>/<organization_name>/<repository_name>:<version_or_sha>` above, that built and pushed the manager image. So am I left to assume that this is for any non-manager images in my operator? Should that be said more explicitly?

Reply:
Also, before you can run this command, you need to initialize the docker buildx driver:
```
 docker buildx create --use
 ```
 
 Lastly, we should probably note that the `.` in the last part of the command is the context path. (Basically where it's going to look for a file called Dockerfile and things referenced in it). You can specificy an alternative filename with the `--file` parameter. It may be worth adding that explicitly to make it obvious how to configure it e.g.:
 `--filename Dockerfile`

Reply:
So, I was very confused about this part of the upstream and kubebuilder docs. 

Is there a chance that I am just duplicating the same steps in slightly different ways? 

If we can leverage kubebuilder and the SDK to do this part, I would love to cut the redundant module.